docs/Configuring-Orion/Using-PostgreSQL.md

```diff
@@ -0,0 +1,59 @@
+description: Storing Information with PostgreSQL 
+<!--- END of page meta data -->
+
+# Store Information with PostgreSQL
+
+PostgreSQL is an object-relational database you can use to store private transaction payloads for Orion. Consider using this option as an aid in [disaster recovery](Disaster-Recovery.md).
+
+## Prerequisites
+
+* [PostgreSQL](https://www.postgresql.org/)
+
+## Set up your PostgreSQL database
+
+In your database:
+
+1. Create the `store` table by using the provided DDL script, as shown in the following example. Substute your own values for `HOST-NAME`, `USER-NAME`, and `DATABASE-NAME`.
+
+    `psql -h <HOST-NAME> -U <USER-NAME> -d <DATABASE-NAME> -f ~/orion/docs/configuring-orion/database/postgres_storage.sql` 
+    
+2. Specify the users for the `store` table and assign permissions to them.
+
+## Configure Orion to use PostgreSQL
+
+In your [Orion configuration file](Configuration-File.md), specify `postgresql` for the `storage` property, as shown in the following example. Substitute your own values for `HOST`, `PORT`, `DATABASE-NAME`, `USER-NAME`, and `PASSWORD`. 
+
+  ```
+  storage="sql:jdbc:postgresql://<HOST>:<PORT>/<DATABASE-NAME>?user=<USER-NAME>&password=<PASSWORD>"
+  ```
+  
+## Test your updated configuration
+
+1. Send the payload to Orion by running the following request:
+
+     ```
+     curl -X POST \
+     http://localhost:8888/send \
+     -H 'Content-Type: application/json' \
+     -d '{ 
+           "payload": "SGVsbG8sIFdvcmxkIQ==",
+           "from": "<KEY-IN-NODEKEY>.PUB",
```

Comment:
there is no mention of .PUB in https://docs.orion.pegasys.tech/en/stable/Getting-Started/Quickstart/ ... shouldn't we use similar content as in quick start here?

Reply:
Let me make sure I understand your comment. Are you suggesting that the doc should say `<OrionPublicKey>` instead of `<KEY-IN-NODEKEY.PUB>`?